ddtrace/tracer/span.go

```diff
@@ -77,6 +77,8 @@ type span struct {
 	noDebugStack bool         `msg:"-"` // disables debug stack traces
 	finished     bool         `msg:"-"` // true if the span has been submitted to a tracer.
 	context      *spanContext `msg:"-"` // span propagation context
+	inProcessor  bool         `msg:"-"` // true if the span is in postProcessor.
+	remove       bool         `msg:"-"` // true if the span should be removed from trace. note: currently does nothing.
```

Comment:
Let's remove this too and let the user work the slice as he pleases.

Reply:
This was addressed in a previous comment, marking it as resolved.